bots/task/test-task

```diff
@@ -131,6 +136,10 @@ class TestTask(unittest.TestCase):
         comment = task.comment(1234, "This is the comment")
         self.assertEqual(comment["body"], "This is the comment")
 
+    def testLabel(self):
+        label = task.label(1234, ['xxx'])
+        self.assertEqual(label, ['xxx'])
```

Comment:
Should also test the default argument and ensure that `bot` comes out, but nitpick..